### ISSUE TITLE
Upload directory is not reading from Media Settings correctly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#210]
```
What steps will reproduce the problem? (Be specific)
1. With CCTM installed, go to Settings --> Media
2. Change the default uploads directory
3. CCTM still expects uploads to be inside wp-content/uploads

There should be a function added that reads this option correctly.
```

Original issue reported on code.google.com by `fireproofsocks` on 11 Oct 2011 at 2:54

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Interesting... wp_upload_dir() is already being used, but changing the media 
directory issues a CCTM warning (it's a WP error, CCTM just passes it on).
```

Original comment by `fireproofsocks` on 12 Oct 2011 at 8:18
- Changed state: **InProgress**

**Comment by GoogleCodeExporter** [comment]
```
Ok, this has been fixed, will be released in 0.9.4.3
Committed revision 450181.
```

Original comment by `fireproofsocks` on 12 Oct 2011 at 8:38
- Changed state: **Fixed**
